spec/src/utils/helpers.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable import/no-unresolved, no-unused-expressions */
+const dotenv = require('dotenv');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const sinonChai = require('sinon-chai');
+const {
+  ourEncodeURIComponent,
+  cleanParams,
+  throwHttpErrorFromResponse,
+  isNil,
+  applyNetworkTimeout,
+} = require('../../../test/utils/helpers'); // eslint-disable-line import/extensions
+const { createAuthHeader } = require('../../../src/utils/helpers');
```

Comment:
This is imported from `src` not `tests`. I think it's the same issue we saw yesterday